components/Member/MemberList.tsx

```diff
@@ -26,25 +26,14 @@ const MemberList = ({ members, isOnInvitationPage }: Props) => {
       <div className="h-full w-full overflow-y-auto card flex flex-col justify-between divide-y">
         <div className="h-fit divide-y">
           {members && members.length ? (
-            members[page - 1].map(({ username, admin_of, member_of }, idx) =>
```

Comment:
admin_of und member_of waren da nicht dabei, aber ich würde sie brauchen damit ich in MemberComponent anzeigen kann ob das ein Admin oder ein Teilnehmer ist. Das sollten wir besprechen und beheben.

Reply:
was heißt die waren da nicht dabei? die gibts schon von beginn an